english/coffee/hollywood/hollywood.md

```diff
@@ -1,3 +1,20 @@
+Marshmellow
+An original screenplay concept by
+Anonymous
+
+Science Fiction:A computer programmer teams up with a unemployed writer to prove his innocence of federal 
+fraud charges.
+
+As the story unfolds, the computer programmer falls in love with a street wise kid who has a time 
+machine which they use to their advantage.
+
+Through hiding the truth until the last moment they manage to break into 1 shopping malls, bring down the 
+dirty politician and win the respect of mob informants. 
+
+Think Matrix meets Jurassic Park.
+
 After writing, what you believe to be to be, a best-selling novel you decided to try to market the book right
 and head to Hollywood. The only choice now is how to get there. You’re afraid of flying and don’t care much for 
-public transportation. 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.